workspace/r/ds-pack/v1/install_packages.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+WORKON_HOME=${WORKON_HOME:-/opt/venv}
+PYTHON_VENV_PATH=${PYTHON_VENV_PATH:-${WORKON_HOME}/reticulate}
+RETICULATE_MINICONDA_ENABLED=${RETICULATE_MINICONDA_ENABLED:-FALSE}
+
+apt-get update && apt-get install -y --no-install-recommends \
+    libpython3-dev \
+    python3-dev \
+    python3-pip \
```

Comment:
Hey @saurabh0402 - Just thinking on why would we need python to be installed in a R workspace 🤔

Reply:
Hey, we do install R in python workspaces and Python in R workspaces. Think this is so that people can easily test things out if needed.